CMakeLists.txt

```diff
@@ -64,14 +64,21 @@ endif()
 include_directories(include ${PROJECT_BINARY_DIR}/include)
 
 configure_file(include/Config.in include/Config.h)
-configure_file(cmake/VecCoreConfig.cmake.in cmake/VecCoreConfig.cmake @ONLY)
+
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+configure_package_config_file(cmake/VecCoreConfig.cmake.in
+  ${CMAKE_CURRENT_BINARY_DIR}/cmake/VecCoreConfig.cmake
+  INSTALL_DESTINATION lib${LIB_SUFFIX}/cmake/${PROJECT_NAME}
+  PATH_VARS CMAKE_INSTALL_INCLUDEDIR CMAKE_INSTALL_DATAROOTDIR
+  )
```

Comment:
The current config file is already relocatable, unless there is some bug.

We can probably use `share/cmake/${PROJECT_NAME}` instead of `lib${LIB_SUFFIX}/cmake/${PROJECT_NAME}`, since VecCore does not depend on the architecture. However, this will still need adjustment for Windows, since ROOT plans to support that as well.

I was thinking about also using `write_basic_package_version_file()` for `VecCoreConfigVersion.cmake`.

Reply:
I considered `write_basic_package_version_file()`, but it seems to add an additional check for the 32/64bit-ness:

``` cmake
# if the installed or the using project don't have CMAKE_SIZEOF_VOID_P set, ignore it:
if("${CMAKE_SIZEOF_VOID_P}" STREQUAL "" OR "8" STREQUAL "")
   return()
endif()

# check that the installed version has the same 32/64bit-ness as the one which is currently searching:
if(NOT CMAKE_SIZEOF_VOID_P STREQUAL "8")
   math(EXPR installedBits "8 * 8")
   set(PACKAGE_VERSION "${PACKAGE_VERSION} (${installedBits}bit)")
   set(PACKAGE_VERSION_UNSUITABLE TRUE)
endif()
```

I don't *think* that's needed since `VecCore` is interface only?

Reply:
If that's the only difference to what I have, then there is no need indeed to use `write_basic_package_version_file()`.